### ISSUE TITLE
[SP2019]Review Electronics Labs

### ISSUE THREAD

**Issue by trickeydan** [sourcebots/tasks#46]

**Comment by Dasepho** [comment]
- [ ] including writing new lab for capacitor sensor (blocked by #74)

**Comment by joe9599** [comment]
capacitive lab no longer needed due to the issues in manufacturing (neither is the metal strip lab) I am currently looking through the labs to review them

**Comment by trickeydan** [comment]
~Anton has agreed to take this on.~

 Deadline is 27th July

**Comment by joe9599** [comment]
The content has already been reviewed and fixed by 2-3 people I believe, I'm just waiting for a chance to review formatting as I don't know how to compile it, the current branch is called something like jmorton  review
